### ISSUE TITLE
RedBlueDoors not trainable with the actual `max_step`

### ISSUE THREAD

**Issue by lcswillems** [maximecb/gym-minigrid#61]
Hi Maxime,

I wasn't able to make the agent learn `RedBlueDoors` and I thought it was because my implementation of agent's memory was buggy but, in fact, it is because the `max_step` is too low.

[In this line](https://github.com/maximecb/gym-minigrid/blob/28f4554056b6b8003123d15bc483b41c9cd6694a/gym_minigrid/envs/redbluedoors.py#L15), I tried `max_steps=20*size*size` instead of `max_steps=10*size*size` and it works now.

**Comment by maximecb** [comment]
But what is 10*size*size here? 360? It seems it really should be trainable with such a high max_steps. Possibly, a different set of hyper-parameters would work.

**Comment by lcswillems** [comment]
I don't know if it should be trainable or not with such max_step. The only thing I can do is to experiment. And actually, I can only say that I wasn't able to make it learn...

If you know hyper-parameters that work, let me know. But I didn't find.

It would be great to know that the environment proposed in your repo is learnable (without curriculum).

**Comment by maximecb** [comment]
The question I would ask is: how often does an untrained agent succeed (percentage of episodes) in getting a reward. It has to be possible to succeed by chance within 360 steps. If the agent succeeds at all, then some learning should ideally take place, but the current hyper parameters likely don't work when the agent doesn't succeed often enough.

**Comment by lcswillems** [comment]
You are right, it is possible to get reward within 360 steps by acting randomly. However, it can't make it learn, even by testing other hyperparameters.

**Comment by maximecb** [comment]
I feel like that's kind of a flaw with PPO, but ok, let's increase max_steps.
